src/server/scripts/Northrend/Nexus/EyeOfEternity/boss_malygos.cpp

```diff
@@ -385,6 +385,9 @@ class boss_malygos : public CreatureScript
             SetPhase(PHASE_NOT_STARTED, true);
             me->SetReactState(REACT_PASSIVE);
             instance->DoStopTimedAchievement(ACHIEVEMENT_TIMED_TYPE_EVENT, ACHIEV_TIMED_START_EVENT);
+            
```

Comment:
This line contains invisible whitespace characters, as pointed out by **ci/circleci: pch_and_sql** .
```suggestion

```